CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeViewPane.java

```diff
@@ -150,7 +158,36 @@ public void handle(ActionEvent e) {
                 }
             }
             leftVBox.setAlignment(Pos.TOP_CENTER);
-
+            
+            final HBox lowerLeftHBox = new HBox();
+            lowerLeftHBox.setPadding(new Insets(30, 10, 10, 20));
+            
+            // Create a checkbox to change users preference regarding showing the Tutorial Page on startup 
+            final Preferences prefs = NbPreferences.forModule(ApplicationPreferenceKeys.class);
+            final CheckBox showOnStartUpCheckBox = new CheckBox("Show on Startup");
+            showOnStartUpCheckBox.setFont(new Font("Arial", 16));
+            lowerLeftHBox.getChildren().add(showOnStartUpCheckBox);
+           
+            showOnStartUpCheckBox.selectedProperty().addListener(new ChangeListener<Boolean>() {
+                @Override
+                public void changed(ObservableValue<? extends Boolean> ov,
+                        Boolean oldVal, Boolean newVal) {
+                    prefs.putBoolean(ApplicationPreferenceKeys.WELCOME_ON_STARTUP, newVal);                     
+                }
+            });        
```

Comment:
should be able to make this a lambda (remove the types of the parameters when you do)